src/docs/release-notes/R25/Summary.mdx

```diff
@@ -0,0 +1 @@
+Migrated all our [Angular](https://brightlayer-ui-components.github.io/angular) and [React](https://brightlayer-ui-components.github.io/react) component developer documentation from Storybook to a new site with powerful features such as Examples page that provide usage context and Playground pages where you can customize components and generate code snippets; added support for React Native v0.70; added [icon style guide](/style/icon-guidelines).
```

Comment:
Since this is the summary, it doesn't have to be as detailed:
```suggestion
Migrated all our [Angular](https://brightlayer-ui-components.github.io/angular) and [React](https://brightlayer-ui-components.github.io/react) component documentation from Storybook to an in-house site with powerful new features, added support for React Native v0.70.x, and added an [icon style guide](/style/icon-guidelines).
```